docs/en/platform-engineers/terraform.md

```diff
@@ -10,10 +10,10 @@ These cloud services are provided by Terraform.
 
 <details>
 
-Download the latest chart, like `terraform-controller-chart-0.1.4.tgz`, from the latest [releases list](https://github.com/oam-dev/terraform-controller/releases) and install it.
+Download the latest chart, like `terraform-controller-chart-0.1.7.tgz`, from the latest [releases list](https://github.com/oam-dev/terraform-controller/releases) and install it.
```

Comment:
Could you please update the version to `0.1.8`? I built the chart based on your latest `supporting GCP` feature https://github.com/oam-dev/terraform-controller/releases/tag/v0.1.8.

Reply:
Ok perfect. I will update this